rancher/v1.2/en/quick-start-guide/index.md

```diff
@@ -31,12 +31,12 @@ If you are interested in trying one of our latest development builds which will
 All you need is one command to launch Rancher server. After launching the container, we'll tail the logs to see when the server is up and running.
 
 ```bash
-$ sudo docker run -d --restart=always -p 8080:8080 rancher/server
+$ sudo docker run -d --restart=always --name rancher-server -p 8080:8080 rancher/server
```

Comment:
Then maybe it would be good to point out that the command:

```bash
sudo docker logs -f containerid
```

would have to be updated with the correct `containerid`, and probably specify how to get that ID. 

Would you like / prefer a combination of `docker ps`, `grep` and `sed` or `awk`, etc to get that correct `containerid`?